pkgs/tools/misc/crypto-tracker/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "crypto-tracker";
+  version = "0.1.8";
```

Comment:
Also noticed this version number is different from the PR title, either this line in the package definition, or the PR title and commit summary line should be changed to the version.
```nix
  version = "0.1.8";
```